### PR TITLE
feature: account hint section #5669

### LINKED ISSUE #5669: [FEATURE] hints section of features to use on your profile
### Description

Many people only add links to their profile because they are unaware of the other features like milestones, testimonials and events etc

When the [PR for forms](https://github.com/EddieHubCommunity/LinkFree/pull/6998) is completed, there will be a sub navigation, so this can have its own page of "profile suggestions" (arrow in screenshot below)

240759736-e532b3d6-5a35-430e-b95a-7d5956b3880e

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Landouda**
## Fixes Issue

Closes #5669

## Changes proposed

Implementation of the hint section feature below the progress bar where a code snippet and a brief description is provided for all missing pofile sections.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

image

## Note to reviewers

I created mdx files inside the components folder to include code snippets inside the hint component and included them in the statistics page. I have doubts about this implementation and I am open to fix it.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6037"></a>

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Thank you for your contribution 👍 sorry for my slow reply, I am working through notifications

The way you did the `mdx` components is interesting, I am thinking of other alternatives of how to achieve this

File: pages/account/statistics.js [hunk 1]
```diff
@@ -113,6 +123,17 @@ export default function Statistics({ data, profile, progress }) {
...
+    "links": { value: "links", link: "https://linkfree.eddiehub.io/docs/how-to-guides/links", description: "Let people discover all your great content in one place.", component: Links },
+    "milestones": { value: "milestones", link: "https://linkfree.eddiehub.io/docs/how-to-guides/milestones", description: "Demonstrate the highlights of your career.", component: Milestones },
+    "tags": { value: "tags", link: "https://linkfree.eddiehub.io/docs/how-to-guides/tags", description: "These are searchable keywords for people to discover your Profile.", component: Tags },
+    "socials": { value: "socials", link: "https://linkfree.eddiehub.io/docs/how-to-guides/socials-shortcuts", description: "Add a shortcut to your favourite social media accounts.", component: Socials },
+    "location": { value: "location", link: "https://linkfree.eddiehub.io/docs/how-to-guides/events", description: "Add events you either hosted or attended.", component: Location },
+    "testimonials": { value: "testimonials", link: "https://linkfree.eddiehub.io/docs/how-to-guides/testimonials", description: "Show off the great feedback you have received.", component: Testimonials },
+  }
```
Comment (eddiejaoude) [inline pages/account/statistics.js:134]: Please check you have VScode Prettier installed, so the formatting automatically matches the projects

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Great work on this Pull Request 👍 I left some inline comments

**Comment by ChinmayMhatre** [comment]
Hey @Landouda, just wanted to check in about progress of the changes, do let us know if you need any help.

**Comment by SaraJaoude** [comment]
@Landouda thank you for your contribution so far.  Due to lack of activity I will be closing this Pull Request and unassigning the Issue so that it does not remain blocked.

**Pull request closed without merging**